examples/webgpu_parallax_uv.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgpu - parallax uv</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<link type="text/css" rel="stylesheet" href="main.css">
+	</head>
+	<body>
+
+		<div id="info">
+			<a href="https://threejs.org" target="_blank" rel="noopener">three.js</a> webgpu - parallax uv<br />
+			Textures by <a href="https://ambientcg.com/view?id=Ice002" target="_blank" rel="noopener">ambientCG</a><br />
+			
+		</div>
+
+		<script type="importmap">
+			{
+				"imports": {
+					"three": "../build/three.module.js",
+					"three/addons/": "./jsm/",
+					"three/nodes": "./jsm/nodes/Nodes.js"
+				}
+			}
+		</script>
+
+		<script type="module">
+
+			import * as THREE from 'three';
+			import { MeshStandardNodeMaterial, texture, parallaxUV, uv } from 'three/nodes';
+
+			import WebGPURenderer from 'three/addons/renderers/webgpu/WebGPURenderer.js';
+
+			import { OrbitControls } from 'three/addons/controls/OrbitControls.js';
+
+			let camera, scene, renderer;
+
+			let controls;
+
+			init();
+
+			function init() {
+
+				// scene
+
+				scene = new THREE.Scene();
+
+				// camera
+
+				camera = new THREE.PerspectiveCamera( 45, window.innerWidth / window.innerHeight, .1, 50 );
+				camera.position.set( 10, 14, 10 );
+
+				// environment
+
+				const environmentTexture = new THREE.CubeTextureLoader()
+					.setPath( './textures/cube/Park2/' )
+					.load( [ 'posx.jpg', 'negx.jpg', 'posy.jpg', 'negy.jpg', 'posz.jpg', 'negz.jpg' ] );
+
+
+				scene.environment = environmentTexture;
+				scene.background = environmentTexture;
+
+				// textures
+
+				const loader = new THREE.TextureLoader();
+
+				const topTexture = loader.load( 'textures/ambientcg/Ice002_1K-JPG_Color.jpg' );
+				topTexture.colorSpace = THREE.SRGBColorSpace;
+
+				const roughnessTexture = loader.load( 'textures/ambientcg/Ice002_1K-JPG_Roughness.jpg' );
+				roughnessTexture.colorSpace = THREE.SRGBColorSpace;
+
+				const normalTexture = loader.load( 'textures/ambientcg/Ice002_1K-JPG_NormalGL.jpg' );
+				normalTexture.colorSpace = THREE.NoColorSpace;
+
+				const displaceTexture = loader.load( 'textures/ambientcg/Ice002_1K-JPG_Displacement.jpg' );
+				displaceTexture.colorSpace = THREE.SRGBColorSpace;
```

Comment:
`roughness` and `displacement` should be `NoColorSpace`.

And likewise, the normal texture, but its color profile is `sRGB`. Regardless, I'd leave it as `NoColorSpace`.

/ping @donmccurdy for second opinion.

Reply:
Agreed, we should use `NoColorSpace` for all textures containing non-color data, including all three here. The color profiles tend to be unreliable, and I'm not sure there's any way to tag an image clearly identifying it as non-color data. If the material looks incorrect when non-color textures are annotated with `NoColorSpace`, then we would need fix the textures.